Excersises/exercise6.js

```diff
@@ -0,0 +1,18 @@
+/* Write a JavaScript function to check if a given string is a palindrome (reads the same forwards and backwards).*/
+
+const checkPalindrome = (str) => {
+  let righToleft = "";
```

Comment:
Please refactor and rename it to (e.g: reversedStr)

Reply:
Ok, done